stablehlo/conversions/linalg/tests/random.mlir

```diff
@@ -22,671 +22,3 @@ func.func @rng_uniform_1d(%min: tensor<f32>, %max: tensor<f32>) -> tensor<10xf32
 // CHECK-NEXT:   linalg.yield %[[VAL5]] : f32
 // CHECK-NEXT: -> tensor<10xf32>
 
-// -----
-
-// CHECK-LABEL: func @rng_uniform_2d
-func.func @rng_uniform_2d(%min: tensor<f32>, %max: tensor<f32>) -> tensor<3x3xf32> {
```

Comment:
Why are all these tests deleted? Did we change the semantics of `stablehlo.rng` with this fix?